### DISCUSSION TITLE
Pages/News with PromotedState:1 do not show in results anymore

### DISCUSSION THREAD

**Discussion by MirasMNC** [microsoft-search/pnp-modern-search#3909] [Q&A] [answered]
We use webpart for displaying news with our custom design and have two interfaces. 

One interface is for endusers with easy query e.g. "{searchTerms} Path:https://server.sharepoint.com/sites/MNOTestSearch **PromotedState:2**" - works well.

Another one is for editors where they can see draft results before publishing to check the look (pictures, title and so on)
"{searchTerms} Path:https://server.sharepoint.com/sites/MNOTestSearch **PromotedState:1**". Unfortunatelly this query stopped working and editors can not see results anymore.

Do you know if some settings was changed? Do you have any advice how to fix it?

**Comment by kasperbolarsen** [comment, marked as answer]
One possibility is that the content isn't indexed. Can you verify that? ( I wrote a guide about how to do that here https://www.m365thinking.com/post/what-to-do-when-the-sharepoint-search-doesn-t-pick-up-your-content-part-1-of-4 )

**Reply by MirasMNC** [reply]
Thank you for your fast response. I verified the index and you are right, I can not see this page with promotedstate:1 in index.
